doc/user/content/sql/create-source/_index.md

```diff
@@ -1,44 +1,231 @@
-Sources](../../overview/key-concepts/#sources).
+## Connectors
 
-## Types of sources
+Materialize bundles **native connectors** for the following external systems:
 
-Materialize can connect to many different external sources of data, each with
-their own requirements. For details about creating sources, view the
-documentation for the type of data you are trying to load into Materialize:
+{{< multilinkbox >}}
+{{< linkbox title="Message Brokers" >}}
+- [Kafka](/sql/create-source/kafka)
+- [Redpanda](/sql/create-source/kafka)
+- [Kinesis Data Streams](/sql/create-source/kinesis)
+{{</ linkbox >}}
+{{< linkbox title="Object Storage" >}}
+- [S3](/sql/create-source/s3)
+{{</ linkbox >}}
+{{< linkbox title="Databases (CDC)" >}}
+- [PostgreSQL](/sql/create-source/postgres)
+{{</ linkbox >}}
+{{< linkbox title="Other" >}}
+- [PubNub](/sql/create-source/pubnub)
+{{</ linkbox >}}
+{{</ multilinkbox >}}
 
-Source type     | Avro                         | Text/bytes                             | Protobuf                                 | CSV                            | JSON
-----------------|------------------------------|----------------------------------------|------------------------------------------|--------------------------------|---------------------------------
-Kafka           | [Avro + Kafka](./avro-kafka) | [Text/bytes + Kafka](./text-kafka)     | [Protobuf + Kafka](./protobuf-kafka)     | [CSV + Kafka](./csv-kafka)     | [JSON + Kafka](./json-kafka)
-Kinesis         | -                            | [Text/bytes + Kinesis](./text-kinesis) | [Protobuf + Kinesis](./protobuf-kinesis) | [CSV + Kinesis](./csv-kinesis) | [JSON + Kinesis](./json-kinesis)
-S3              | -                            | [Text/bytes + S3](./text-s3)           | -                                        | [CSV + S3](./csv-s3)           | [JSON + S3](./json-s3)
-PubNub          | -                            | [Text + PubNub](./text-pubnub)         | -                                        | -                              | [JSON + PubNub](./json-pubnub)
-[Postgres](./postgres)  | -  | - | -  | -  | -
+For details on the syntax, supported formats and features of each connector, check out the dedicated `CREATE SOURCE` documentation pages.
 
+## Formats
 
-Don't see what you're looking for? [Let us know on GitHub](https://github.com/MaterializeInc/materialize/issues/new?labels=C-feature&template=feature.md).
+To read from an external data source, Materialize must be able to determine how to decode raw bytes from different formats into data structures it can understand at runtime. This is handled by specifying a `FORMAT` in the `CREATE SOURCE` statement.
+
+### Avro
+
+<p style="font-size:14px"><b>Syntax:</b> <code>FORMAT AVRO</code></p>
+
+Materialize can decode Avro messages by integrating with a schema registry to retrieve a schema, and automatically define the columns and data types to use in the source.
+
+##### Schema versioning
```

Comment:
I did, because H4 looks an awful lot like H3, which made the page confusing. :marta: But at some point there was a `#### Format details` header preceding these; I just...didn't like how unnecessary it looked.